spec/models/tattoo_spec.rb

```diff
@@ -3,7 +3,7 @@
 RSpec.describe Tattoo, type: :model do
 
   context 'tattoos' do
-    let!(:tattoo) { FactoryGirl.create(:tattoo) }
+    let!(:tattoo) { FactoryGirl.create(:tattoo, :title => 'Badass Celtic Armband') }
```

Comment:
By "the new Ruby 1.9 hash syntax", Hound means stuff like `title: "..."` instead of `:title => "..."`